### PR TITLE
Re-add deprecation page + fix modal + cleanup

### PR DESCRIPTION
**Pull request by antograssiot**
This PR add the following:
 
### 1) header navigation bar style update to be closer to the book

fromcapture d ecran 2016-07-17 a 08 31 21 to
capture d ecran 2016-07-17 a 08 32 41

### 2) add a deprecation page
capture d ecran 2016-07-17 a 08 34 58

### 3) fix the modal
Except the version buttons, the others were not clickable and the layout was kind of broken
from
capture d ecran 2016-07-17 a 08 36 49
to
capture d ecran 2016-07-17 a 08 37 35

### 4) remove the old example template folders (default and bootstrap) and update the readme
we got the updated one from composer and I would even remove them from the composer.json personally as they don't bring much help...

### CODE REVIEW

**Comment by markstory** [comment]
Does the wider navbar work at the various breakpoints? I remember when I was initially building the updated API theme I had a bunch of problems with overlaps and double tall headers.

File: templates/cakephp/resources/css/styles.css [hunk 1]
```diff
@@ -1815,9 +1819,60 @@ header {
...
+.modal .btn-download {
+    margin-top: 20px;
+}
+
+.t-api-nav-el {
+    display: inline-block;
+    float: left;
+    color: #bdbdb5;
+    margin: 0 !important;
+    text-transform: uppercase;
```
Comment (markstory) [inline templates/cakephp/resources/css/styles.css]: I'm not sure the upper case looks good here. Even if it is more consistent.

**Comment by markstory** [comment]
Thank you for removing the old templates :clap:

File: templates/cakephp/resources/css/styles.css (hunk 1 above)
Comment (antograssiot) [inline templates/cakephp/resources/css/styles.css]: i'll remove it, no pb

**Comment by antograssiot** [comment]
> Does the wider navbar work at the various breakpoints?
 
yes it does. I saw you add problem and fixes with re-defining the width of each sidebar section.

**Comment by antograssiot** [comment]
capture d ecran 2016-07-17 a 16 47 39

@markstory like that you you also want me to remove the uppercase on "VERSION" ?

**Comment by markstory** [comment]
Hrm. I look at the photoshop files I originally got from the designer and they had:

screen shot 2016-07-17 at 22 02 29

So perhaps the way you had it before was better/more correct. I apologize for going back and forth on this.

**Comment by antograssiot** [comment]
@markstory no pb. Here is the new result just before the breakpoint that switched to the two buttons layout (chen compressed to the maximum in other words)
capture d ecran 2016-07-18 a 07 39 16

**Comment by markstory** [comment]
Great work @antograssiot

**Pull request merged**